sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
+    if count == 0:
+      raise ValueError("count (%s) must be a non-zero natural number." % count)
```

Comment:
<!--thread_id:cc_158152744_t; commit:17604fb4ecfba607daa3c22312c02865b02ecee7; resolved:0-->
<!--section:context-quote-->
> **mariapython** wrote:
> The reason why I didn't check for non-int was that the proto enforced integers, so 2.3 wouldn't be accepted. But the message was not clear, and as you say, it is cleaner to match the error statement, so I modified it per your suggestion.

<!--section:body-->
Could you change there error string to `count (%d) must be a positive integer.` ?